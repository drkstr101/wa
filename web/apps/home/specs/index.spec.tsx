import { render } from '@testing-library/react';
import React from 'react';
import Index from '../pages/index';

describe('Index', () => {
  it('should render successfully', () => {
    const { container } = render(<Index />);
    expect(container).toBeTruthy();
    expect(container).toMatchSnapshot();
  });
});
