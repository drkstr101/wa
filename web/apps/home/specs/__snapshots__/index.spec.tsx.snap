// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render successfully 1`] = `
<div>
  <main
    __self="[object Object]"
    __source="[object Object]"
  >
    <div
      __source="[object Object]"
      class="relative overflow-hidden"
    >
      <div
        __source="[object Object]"
        class="pt-10 sm:pt-16 lg:pt-8 lg:pb-14 lg:overflow-hidden"
      >
        <div
          __source="[object Object]"
          class="mx-auto max-w-7xl lg:px-8"
        >
          <div
            __source="[object Object]"
            class="lg:grid lg:grid-cols-2 lg:gap-8"
          >
            <div
              __source="[object Object]"
              class="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 sm:text-center lg:px-0 lg:text-left lg:flex lg:items-center"
            >
              <div
                __source="[object Object]"
                class="lg:py-24"
              >
                <a
                  __source="[object Object]"
                  class="inline-flex items-center text-white bg-base-300 rounded-full p-1 pr-2 sm:text-base lg:text-sm xl:text-base hover:text-neutral-content"
                  href="#"
                >
                  <span
                    __source="[object Object]"
                    class="px-3 py-0.5 text-white text-xs font-semibold leading-5 uppercase tracking-wide bg-accent rounded-full"
                  >
                    We're hiring
                  </span>
                  <span
                    __source="[object Object]"
                    class="ml-4 text-sm"
                  >
                    Visit our careers page
                  </span>
                  <svg
                    __source="[object Object]"
                    aria-hidden="true"
                    class="ml-2 w-5 h-5 text-primary"
                    fill="currentColor"
                    viewBox="0 0 20 20"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </a>
                <h1
                  __source="[object Object]"
                  class="mt-4 text-4xl tracking-tight font-extrabold text-white sm:mt-5 sm:text-6xl lg:mt-6 xl:text-6xl"
                >
                  <span
                    __source="[object Object]"
                    class="block"
                  >
                    A better way to
                  </span>
                  <span
                    __source="[object Object]"
                    class="block text-accent"
                  >
                    ship web apps
                  </span>
                </h1>
                <p
                  __source="[object Object]"
                  class="mt-3 text-base text-gray-300 sm:mt-5 sm:text-xl lg:text-lg xl:text-xl"
                >
                  Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui Lorem cupidatat commodo. Elit sunt amet fugiat veniam occaecat fugiat.
                </p>
                <div
                  __source="[object Object]"
                  class="mt-10 sm:mt-12"
                >
                  <form
                    __source="[object Object]"
                    action="#"
                    class="sm:max-w-xl sm:mx-auto lg:mx-0"
                  >
                    <div
                      __source="[object Object]"
                      class="sm:flex"
                    >
                      <div
                        __source="[object Object]"
                        class="min-w-0 flex-1"
                      >
                        <label
                          __source="[object Object]"
                          class="sr-only"
                          for="email"
                        >
                          Email address
                        </label>
                        <input
                          __source="[object Object]"
                          class="block w-full px-4 py-3 rounded-md border-0 text-base text-gray-900 placeholder-gray-500 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-teal-300 focus:ring-offset-gray-900"
                          id="email"
                          placeholder="Enter your email"
                          type="email"
                        />
                      </div>
                      <div
                        __source="[object Object]"
                        class="mt-3 sm:mt-0 sm:ml-3"
                      >
                        <button
                          __source="[object Object]"
                          class="block w-full py-3 px-4 rounded-md shadow bg-accent text-white font-medium hover:bg-teal-600 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-teal-300 focus:ring-offset-gray-900"
                          type="submit"
                        >
                          Start free trial
                        </button>
                      </div>
                    </div>
                    <p
                      __source="[object Object]"
                      class="mt-3 text-sm text-gray-300 sm:mt-4"
                    >
                      Start your free 14-day trial, no credit card necessary. By providing your email, you agree to our
                       
                      <a
                        __source="[object Object]"
                        class="font-medium text-white"
                        href="#"
                      >
                        terms of service
                      </a>
                      .
                    </p>
                  </form>
                </div>
              </div>
            </div>
            <div
              __source="[object Object]"
              class="mt-12 -mb-16 sm:-mb-48 lg:m-0 lg:relative"
            >
              <div
                __source="[object Object]"
                class="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 lg:max-w-none lg:px-0"
              >
                <img
                  __source="[object Object]"
                  alt=""
                  class="w-full lg:absolute lg:inset-y-0 lg:left-0 lg:h-full lg:w-auto lg:max-w-none"
                  src="/images/cloud-illustration-indigo-400.svg"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </main>
</div>
`;
